test/data-model.mjs

```diff
@@ -46,12 +39,28 @@ process.argv.forEach((val, index) => {
   }
 });
 
-// load the annotationMusts test list
-const musts = JSON.parse(
-  fs.readFileSync(
-    'node_modules/web-annotation-tests/annotations/annotationMusts.test'
-  )
-);
+function readSchema(schemaPath, base = 'web-annotation-tests/') {
+  const resolverOptions = { extensions: ['.json', '.test'] };
+  const resolvedPath = resolve.sync(`${base}${schemaPath}`, resolverOptions);
+  const schemaUnparsed = fs.readFileSync(resolvedPath);
+  return JSON.parse(schemaUnparsed);
+}
+
+const DEFINITIONS = [
+  'annotations',
+  'bodyTarget',
+  'choiceSet',
+  'collections',
+  'id',
+  'otherProperties',
+  'specificResource',
+].map(name => readSchema(`definitions/${name}`));
```

Comment:
Is persisting the names in the code here better than picking them up from the file names? Not opposed to this approach, just not sure what it's solving for exactly. 😃

Reply:
I was trying not to assume the location of the `web-annotation-tests` package on disk, so I moved things to use `resolve`. However, `resolve` won't easily resolve a directory. As far as I can tell, it's really only good for resolving files. There are other, less ubiquitous packages I might have pulled in for that purpose, but it felt like overkill for something that won't change much. In the end, this felt concise and readable to me.